.php_cs.dist

```diff
@@ -1,7 +1,7 @@
 <?php
 
 $finder = new PhpCsFixer\Finder();
-$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
```

Comment:
The `$description` argument was removed in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2244.

Reply:
Why have you also removed `$name`, when only `$description` has been deprecated?